### ISSUE TITLE
CWGame does not return the same amount of games as BGame

### ISSUE THREAD

**Issue by jrhessey** [chadwickbureau/retrosheet#35]
When running the default script for cwgame vs bgame, I do not get all of the games.  I've found that I'm missing the at least the last 3 Cincinnati games of the year against San Francisco.  I've attached the game files from each of these commands as well.  Is there something I am doing wrong?  Thanks for any assistance.

cwgame -y 1971 1971*.ev* > game1971.csv

vs.

bgame -y 1971 1971*.ev* > game1971.csv

[bgame1971.csv](https://github.com/user-attachments/files/17670555/bgame1971.csv)
[cwgame1971.csv](https://github.com/user-attachments/files/17670556/cwgame1971.csv)

**Comment by tturocy** [comment]
In order to reproduce a problem, it's necessary to provide all of the inputs.  Without the original input files, this is impossible to address.

**Comment by jrhessey** [comment]
I apologize if I posted this to the wrong areaI thought I was in the right area until about 10 minutes ago.  lol!  

I'm using the zip files directly from retrosheet, not your retrosheet files.

[1971eve.zip](https://github.com/user-attachments/files/17671961/1971eve.zip)

**Comment by tturocy** [comment]
We have just run `cwgame` on these files and got the correct output, so we cannot reproduce this.

Also, the bgame1971.csv output attached is obviously incorrect - columns are shifted.  So `BGAME` is not producing correct output.

When we have seen things like this in the past it usually means something is corrupted in the source files on yours.  It's definitely not a Chadwick problem as far as we can see.

**Comment by jrhessey** [comment]
OK.  I did download the zip file a couple more times from the site and from this post and got the same results, 1911 games and missing the final 27 Cincinnati games from 8/8/1971 to the end of the season.  I did look at the data and notice there was a couple 99#'s on 8/7 that only exists in that game.  I did run the cwgame on the event file again after removing the pound sign leaving the 99 for those records in the 1971CIN.evn and ended up with 1938 games and the missing 27 games are now in there.

![image](https://github.com/user-attachments/assets/464723bc-cc4b-47f5-a7ec-a12789fadf2b)
